### DISCUSSION TITLE
How to debug teal code?

### DISCUSSION THREAD

**Discussion by acodervic** [teal-language/tl#745] [Q&A]
I can debug lua vode with vscode. And how to debug teal code?
Is there has any way to do it? 
I want to debug teal code directly.
Thanks everyone!

**Comment by hishamhm** [comment]
I'm not a vscode user myself, but perhaps @pdesaulniers might have some info on how his plugin can interop with other Lua plugins?

**Comment by pdesaulniers** [comment]
Hello!

Some people have managed to use [Local Lua Debugger](https://marketplace.visualstudio.com/items?itemName=tomblind.local-lua-debugger-vscode) for debugging Teal source files. However, you might need to change VSCode's default settings to make it work, since local-lua-debugger does not enable breakpoint support by default for Teal code: https://github.com/teal-language/vscode-teal/issues/62
